tests/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from django.urls import path, include
+
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('accounts/', include('allauth.urls'))
```

Comment:
as explained in chat, this won't help us much, see https://django-rest-auth.readthedocs.io/en/latest/

Reply:
Yes. thanks